pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -3087,6 +3106,17 @@ func fillEntryFromRow(entry *common.RegistrationEntry, r *entryRow) error {
 	if r.TrustDomain.Valid {
 		entry.FederatesWith = append(entry.FederatesWith, r.TrustDomain.String)
 	}
+
+	// Default to X509SvidTTL database value. If not found then try TTL database value
+	if r.RegX509SvidTTL.Valid {
+		entry.X509SvidTtl = int32(r.RegX509SvidTTL.Int64)
+	} else if r.RegTTL.Valid {
+		entry.X509SvidTtl = int32(r.RegTTL.Int64)
+	}
```

Comment:
This check does not function as expected. Due to the way GORM works, the x509-svid-ttl field has already been set on all entries that were created since the column was introduced. This means that for these entries, .RegX509SvidTTL is valid, with a zero value.

This all makes me think that we should probably consider just using the `ttl` column as the  X509-SVID-TTL value and drop the new (and unused) `x509_svid_ttl` column... that might ease some of our downgrade compat concerns as well. I'll bring this up with the other maintainers.

Reply:
We discussed this and the general concensus is that we should forgo using the new x509_svid_ttl column (and remove it in 1.6.0) and just use the existing `ttl` column as the x509 svid ttl column.

Reply:
Ok, that sounds good. I may not be able to get that in tonight, but will try.

Reply:
I was able to make this change as well. It wasn't as large as I was expecting. And I agree, this is the better way to go.

Reply:
Excellent!